app/services/notifications_list.rb

```diff
@@ -8,12 +17,20 @@ class NotificationsList
   }.freeze
 
   def self.for(application_choice, include_ratifying_provider: false, event: nil)
-    notification_name = NOTIFICATION_PREFERENCE_NAMES_FOR_EVENTS.select { |k, v| k if event.in? v }.keys.first
+    notification_name = NotificationsList.feature_flag_preference_names.select { |k, v| k if event.in? v }.keys.first
```

Comment:
```suggestion
    notification_name = self.feature_flag_preference_names.select { |k, v| k if event.in? v }.keys.first
```

Reply:
Sorted, as it's a class method it can find other class methods without the extra scoping!